src/richie/apps/courses/management/commands/migrate_course_run_resource_link.py

```diff
@@ -0,0 +1,62 @@
+"""Command migratind richie course_runs course_link to joanie uri."""
+
+import logging
+from http import HTTPStatus
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+import requests
+from requests.exceptions import RequestException
+
+from richie.apps.courses.models import CourseRun
+
+logger = logging.getLogger("richie.commands.core.migrate_course_run_course_link")
+
+
+class Command(BaseCommand):
+    """
+    Migrate course run course link from edx to joanie uri.
+    """
+
+    help = __doc__
+
+    def handle(self, *args, **options):
+        """
+        Handle the command migrating existing course run with the course link return by joanie.
+        """
+        joanie_backend = settings.JOANIE_BACKEND
+        for course_run in CourseRun.objects.all():
```

Comment:
Just for memory optimization, I'll use `iterator` instead of `all`.

```suggestion
        for course_run in CourseRun.objects.iterator():
```